### PR TITLE
some minor clean up, adam please take a look and see if you can take …some of my changes

### PR DESCRIPTION
**Pull request by petertbernhardt**
need to update the key codes in the if statement

### CODE REVIEW

File: views/components/UpgradePanelHeader.tsx [hunk 1]
```diff
@@ -45,9 +51,12 @@ export default function UpgradePanelHeader() {
             variant="standard"
             className=""
             value={customName}
-            onChange={(e) => {
-              setCustomName(e.target.value);
-              debounceSave(e.target.value);
+            onChange={(e) => saveName(e.target.value)}
+            onKeyPress={(e) => {
+              // if enter or escape key pressed, exit edit mode
+              if (e.keyCode === "" || e.keyCode === "") {
```
Comment (petertbernhardt) [inline views/components/UpgradePanelHeader.tsx:57]: this needs to be finished

**Comment by petertbernhardt** [comment]
whole thing needs to be tested, too!

File: views/components/UpgradePanelHeader.tsx (hunk 1 above)
Comment (petertbernhardt) [reply views/components/UpgradePanelHeader.tsx]: done

**Comment by AdamLay** [comment]
Thank you! Had to change the enter/escape keys to numbers to fix the build.

**Pull request merged**